tests/cases/unit/Repository/PersistenceHelperTest.phpt

```diff
@@ -58,23 +70,32 @@ class PersistenceHelperTest extends TestCase
 		Assert::same(
 			[
 				$translator,
-				$author, $translator->favoredBy,
+				$author,
+				$translator->favoredBy,
```

Comment:
It was intentionally one the same line – every line is a single iteration of for cycle in `getCascadeQueue`.